3rd-party/class.jetpack-amp-support.php

```diff
@@ -420,6 +411,35 @@ public static function amp_disable_sharedaddy_css( $enqueue ) {
 		return $enqueue;
 	}
 
+	/**
+	 * Enqueues the AMP specific sharing styles for the sharing icons.
+	 */
+	public static function amp_enqueue_sharing_css() {
+		if ( self::is_amp_request() ) {
+			wp_enqueue_style( 'sharedaddy-amp', plugin_dir_url( dirname( __FILE__ ) ) . 'modules/sharedaddy/amp-sharing.css', array( 'social-logos' ), JETPACK__VERSION );
+		}
+	}
+
+	/**
+	 * For the AMP Reader mode template, include styles that we need.
+	 */
+	public static function amp_reader_sharing_css() {
+		/*
+		 * We'll need to output the full contents of the 2 files
+		 * in the head on AMP views. We can't rely on regular enqueues here.
+		 *
+		 * phpcs:disable WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+		 * phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
+		 */
+		echo file_get_contents( JETPACK_SOCIAL_LOGOS_DIR . 'social-logos.css' );
```

Comment:
There's a bug on this line which I've opened a fix for: https://github.com/Automattic/jetpack/pull/19503